### ISSUE TITLE
./extract_firmware.sh: Syntaxerror on line 114

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dxskj/firmware-mod-kit#15]
```
unexpected else 

empty section between then and else did not work for me. Adding echo (dummy) 
worked for me.


```

Original issue reported on code.google.com by `fsbkiller` on 13 Dec 2010 at 9:12

**Comment by GoogleCodeExporter** [comment]
```
Hello,
same here debian 6.0.
thx
```

Original comment by `prinz3nr...@gmail.com` on 17 Feb 2011 at 7:31

**Comment by GoogleCodeExporter** [comment]
```
Issue should be fixed; tested fine in Ubuntu 10.04.
```

Original comment by `heffne...@gmail.com` on 30 Aug 2011 at 12:01
- Changed state: **Fixed**
